docs/proposal-subscriptions.md

```diff
@@ -0,0 +1,280 @@
+# Subscription Model Proposal
+
+## The Introduction
+
+Two of the central pillars of the new UI are incremental updates and fine-grained subscriptions.
+
+* Incremental updates involves sending only the data which has changed to the UI, this:
+    * Reduces network traffic.
+    * Reduces the memory footprint of the UI.
+    * Allows us to move away from using a global update loop which makes the UI more responsive.
+* Fine grained subscriptions allow individual Vues (Tree, Graph, Dot, ...) to subscribe to the set o data which they actually need at a particular moment in time, this;
```

Comment:
```suggestion
* Fine grained subscriptions allow individual views (Tree, Graph, Dot, ...) to subscribe to the set o data which they actually need at a particular moment in time, this;
```

Reply:
This was an intentional pun!

May need to change this anyway to help separate Cylc "Views" (e.g. tree, dot, graph) from Vue "views" (i.e. application routes / page model thinggies).

Reply:
Oh, was it ... he heh :grin: